README.md

```diff
@@ -199,5 +222,55 @@ We recommand to use the authenticating mode using the settings.xml. It's more se
 * **jobType**
   - represents the type of job you want to create
   - default value : "java-scala"
-  - for the moment, only "java-scala" works, but maybe in the future, more jobType will be availabe
- 
+  - for the moment, only "java-scala" works, but maybe in the future, more jobType will be available
```

Comment:
By default I prefer not adding sentences like : `but maybe in the future, more jobType will be available`
`for the moment, only "java-scala" works` is enough ;)

Reply:
done